test/models/transforms/test_input.py

```diff
@@ -847,6 +848,88 @@ def test_w_skip_expand(self):
             self.assertTrue(torch.allclose(tf_X, expected_X.expand(3, 5, -1, -1)))
 
 
+class TestAppendFeaturesFromCallable(BotorchTestCase):
+    def test_append_features_from_callable(self):
+        # basic init, provided indices
+        for dtype in [torch.float, torch.double]:
+            tkwargs = {"device": self.device, "dtype": dtype}
+            with self.assertRaises(ValueError):
+                transform = AppendFeaturesFromCallable(
+                    d=2,
+                    indices=[0, 1, 2],
+                    f=lambda x: torch.sum(x, dim=-1, keepdim=True),
```

Comment:
This is not wrong, but let's use the defined order to make it consistent / more readable
```suggestion
                    f=lambda x: torch.sum(x, dim=-1, keepdim=True),
                    d=2,
                    indices=[0, 1, 2],  
```

Reply:
How about defining `lambda x: torch.sum(x, dim=-1, keepdim=True)` as a function in the test and using that (rather than re-defining the lambda function in every call)?

Reply:
Done ;)